home-exercises/src/io/codelex/arithmetic/practice/Exercise2OddEven.java

```diff
@@ -0,0 +1,14 @@
+package io.codelex.arithmetic.practice;
+
+import java.util.Scanner;
+
+public class Exercise2OddEven {
+    public static void main(String[] args) {
+        Scanner n = new Scanner(System.in);
+        System.out.println("Enter the number");
+        int num = n.nextInt();
+        if (num % 2 == 0) {
+            System.out.println("The given number is even");
+        } else System.out.println("The given number is odd");
```

Comment:
IF vienmēr liekam figūriekavas, citur par to nekomentēšu - vienkārši pārej ar acīm pāri citiem uzdevumiem.